src/main/kotlin/xyz/atrius/waystones/commands/WarpstoneCommand.kt

```diff
@@ -0,0 +1,71 @@
+package xyz.atrius.waystones.commands
+
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import org.bukkit.Bukkit
+import xyz.atrius.waystones.utility.defaultWarpKey
+import xyz.atrius.waystones.utility.translateColors
+import xyz.atrius.waystones.utility.pluralS
+
+object WarpstoneCommand : CommandExecutor {
+    // CommandExecutor / Subcommand Manager
+    override fun onCommand(
+        sender: CommandSender,
+        cmd: Command,
+        lbl: String,
```

Comment:
Would stick to full-word variable names, these can easily obfuscate things if you're using these and need to quickly scan over a file.